### ISSUE TITLE
Accuracy is always None in 1.29.1 with Google Geocode

### ISSUE THREAD

**Issue by eely22** [DenisCarriere/geocoder#282]
Once upgrading to 1.29.1, the google geocode accuracy value is always None.

In 1.25:
```
>>> import geocoder
>>> r = geocoder.google("326 Chestnut Hill Rd New Boston, NH 03070", key='XYZ')
>>> r.accuracy
u'ROOFTOP'
```

In 1.29.1:
```
>>> import geocoder
>>> r = geocoder.google("326 Chestnut Hill Rd New Boston, NH 03070", key='XYZ')
>>> r.accuracy
>>> r.accuracy is None
True
```

I would expect the accuracy to return the same in both versions. Unless this was removed? I couldn't find anything in the documentation about this functionality being removed.

**Comment by ebreton** [comment]
Hi @eely22 ,

Yes you are right, it is a bug and the latest version should behave as the first one.

I have pushed a fix in PR #281 , which should fix the issue.

Could you please try the code from this branch `feature_more_providers_upgraded` and confirm that is back ok for you ?

Cheers,
Manu

**Comment by eely22** [comment]
@ebreton Tested the branch and it does seem to fix this specific issue. Thanks!

**Comment by DenisCarriere** [comment]
Published a new released with this fix `v1.29.3`
